book/src/help_vc.md

```diff
@@ -69,6 +69,11 @@ Options:
           this server (e.g., http://localhost:5062).
       --http-port <PORT>
           Set the listen TCP port for the RESTful HTTP API server.
+      --http-token-path <HTTP_TOKEN_PATH>
+          Path to file containing the HTTP API token for validator client
+          authentication. If not specified, defaults to
+          {validators-dir}/api-token.txt. The file will be created if it does
+          not exist.
```

Comment:
Small nitpick, The last sentence is a bit of ambiguous, I think the description is clear enough without the last sentence, hence suggesting to remove it.
```suggestion
          {validators-dir}/api-token.txt. 
```

Reply:
Edit: Need to change in the `cli.rs` too